### ISSUE TITLE
JSDoc Public Items

### ISSUE THREAD

**Issue by mikejr83** [diefarbe/node-lib#18]
Document all the public properties, methods, etc in the library.

**Comment by Whispres** [comment]
Hello, I am interested in helping with documentation. Is this project still active?

**Comment by mikejr83** [comment]
Yep still alive. My day job has been pretty busy lately so I haven't been on. You're more than welcome to fork, make changes, and submit a PR.

Thanks!

**Comment by Whispres** [comment]
I completely understand. Day jobs can be rough. Do you have another preference for how I contact you when I need to ask questions?

**Comment by mikejr83** [comment]
Most of the guys are on Discord: https://discordapp.com/channels/457258397859971094/463873881543933963

I'm on GitHub for my day job so I should get an alert pretty quick ;)
